experiments/reco/tx_adjust_fit.py

```diff
@@ -0,0 +1,71 @@
+# %%
+# Imports
+import numpy as np
+import matplotlib.pyplot as plt
+from scipy.optimize import curve_fit
+
+# %%
+session_name = "2023-11-01-session"
+acquisition = "2023-11-01-093257-tx_adjust_fid"
+
+# session_name = "2023-10-31-session"
+# acquisition = "2023-10-31-172257-tx_adjust_fid"
+
+raw_data = np.load(f"/home/schote01/spcm-console/{session_name}/{acquisition}/raw_data.npy")
```

Comment:
It could be fine if we could eliminate user-specific path definitions

Reply:
Yes, I agree, the problem with these reco scripts is, that we don't have a general data store which is accessible by anyone (I would prefer to keep the data per user). But maybe the best solution would be to keep all reco stuff out of this repository?